README.md

```diff
@@ -105,23 +105,38 @@ service decode2text {
 (...)
 
 ```
-Partial & full parameters : 3 and 20 are the NGram values for header fields, which means the keywords created for fields (To, Cc, ...) are between 3 and 20 chars long.
-Full words are also added by default (if not longer than 245 chars, which is the limit of Xapian capability).
+
+Indexing options
+----------------
+
+| Option         | Description                    | Possible values                      | Default value |
+|----------------|--------------------------------|--------------------------------------|---------------|
+| partial & full | NGram values for header fields | between 3 and 20 characters          | 3 & 20        |
+| verbose        | Logs verbosity                 | 0 (silent), 1 (verbose) or 2 (debug) | 0             |
+| attachments    | Index attachments or not       | 0 or 1                               | 0             |
```

Comment:
@arodier "Attachment" has been removed from last version. The trigger is just if fts_decoder option is set or not

Reply:
Thanks, I will remove it.